src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -50,10 +51,53 @@
     private static final String GROUP_COMPRESSION = "File Compression";
     private static final String DEFAULT_FILENAME_TEMPLATE = "{{topic}}-{{partition}}-{{start_offset}}";
 
+    private static final String GROUP_RETRY_BACKOFF_POLICY = "Retry backoff policy";
+    public static final String KAFKA_RETRY_BACKOFF_MS_CONFIG = "kafka.retry.backoff.ms";
+
     protected AivenCommonConfig(final ConfigDef definition, final Map<?, ?> originals) {
         super(definition, originals);
     }
 
+    protected static void addKafkaBackoffPolicy(final ConfigDef configDef) {
+        configDef.define(
+                KAFKA_RETRY_BACKOFF_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                null,
+                new ConfigDef.Validator() {
+
+                    static final long MAXIMUM_BACKOFF_POLICY = 86400000; // 24 hours
```

Comment:
Why long? I thing int should be enough, no?

Reply:
long is a default for MS in java

Reply:
It is not that bad anyway.